app/services/slack_service.rb

```diff
@@ -15,11 +15,11 @@ def get_slack_users
     all_members
   end
 
-  def  notify_user(user_name, user_id)
+  def  notify_user(user_name, user_id, message)
```

Comment:
Unused method argument - user_name. If it's necessary, use _ or _user_name as an argument name to indicate that it won't be used.